File: 프로그래머스/2/42578. 의상/의상.py
def solution(clothes):
    answer = 1
    dic = {}
    for cloth in clothes:
        if cloth[1] in dic:
            dic[cloth[1]] += 1
        else:
            dic[cloth[1]] = 1
    print(dic)
    for d in dic:
        answer *= (dic[d]+1)
        print(dic[d])
    return answer -1

File: 프로그래머스/2/42578. 의상/의상.js
function solution(clothes) {
    dic = {}
    for (cloth of clothes) {
        if (dic[cloth[1]]) {
            dic[cloth[1]] += 1
        } else {
            dic[cloth[1]] = 1
        }
    }
    console.log(dic)
    var answer = 1;
    for (cnt in dic) {
        answer *= (dic[cnt] + 1)
    }
    return answer - 1;
}